docs/sources/operators-guide/reference-http-api/index.md

```diff
@@ -705,54 +705,6 @@ rules:
       severity: warning
 ```
 
-#### Federated rule groups
-
-A federated rule groups is a rule group with a non-empty `source_tenants`.
-
-The `source_tenants` field allows aggregating data from multiple tenants while evaluating a rule group. The expressions
-of each rule in the group will be evaluated against the data of all tenants in `source_tenants`. If `source_tenants` is
-empty or omitted, then the tenant under which the group is created will be treated as the `source_tenant`.
-
-Federated rule groups are skipped during evaluation by default. This feature depends on
-the cross-tenant query federation feature. To enable federated rules
-set `-ruler.tenant-federation.enabled=true` and `-tenant-federation.enabled=true` CLI flags (or their respective YAML
-config options).
-
-During evaluation query limits applied to single tenants are also applied to each query in the rule group. For example,
-if `tenant-a` has a federated rule group with `source_tenants: [tenant-b, tenant-c]`, then query limits for `tenant-b`
-and `tenant-c` will be applied. If any of these limits is exceeded, the whole evaluation will fail. No partial results
-will be saved. The same "no partial results" guarantee applies to queries failing for other reasons (e.g. ingester
-unavailability).
-
-The time series used during evaluation of federated rules will have the `__tenant_id__` label, similar to how it is
-present on series returned with cross-tenant query federation.
-
-**Considerations:** Federated rule groups allow data from multiple source tenants to be written into a single
-destination tenant. This makes the existing separation of tenants' data less clear. For example, `tenant-a` has a
-federated rule group that aggregates over `tenant-b`'s data (e.g. `sum(metric_b)`) and writes the result back
-into `tenant-a`'s storage (e.g. as metric `sum:metric_b`). Now part of `tenant-b`'s data is copied to `tenant-a` (albeit
-aggregated). Have this in mind when configuring the access control layer in front of mimir and when enabling federated
-rules via `-ruler.tenant-federation.enabled`.
-
-#### Example "federated rules group" request body
-
-```yaml
-name: <string>
-interval: <duration;optional>
-source_tenants:
```

Comment:
I think was very useful to show an example, so I would keep it. Since you've moved it outside the HTTP API, instead of calling it an example of "request body" you could just say it's example of a rule group configuration. Also, instead of the spec, I would actually show a real example (no need to show every single field, just give an idea where `source_tenants` field is), like:

```
name: MyGroupName
source_tenants: ["tenant-a", "tenant-b"]
rules:
  - record: sum:metric
    expr: sum(metric)
```

Reply:
applied @ 3e4464783b149ee86985d8c4a4e425f3b38b6e6d